packages/big-design/src/managers/alerts/manager.ts

```diff
@@ -84,8 +81,27 @@ class AlertsManager {
     };
   };
 
+  private afterEvent() {
+    this.manageTimeout();
+    this.notifySubscribers();
+  }
+
+  private manageTimeout() {
+    const alert = this.alerts[0] ?? null;
+
+    if (this.timeout) {
```

Comment:
Won't this reset the timer for an ongoing `autoDismiss`?

Eg: `Alert 1` is displayed with `autoDismiss`, 4 seconds go by and a new alert is is added. `Alert 1` will remain for 9 seconds now. Or am I missing something?

Reply:
Good gatch, I assumed that newly added alert would replace the current alert. Let me think about how to alleviate this.